### ISSUE TITLE
I found that the val_loss can not converge while the train_loss can converge

### ISSUE THREAD

**Issue by whytin** [Superlee506/Mask_RCNN#17]
Thank you first for your nice work! I use this model to train the new datasets (Penn Action), and I found  that the val_loss can not converge while the train_loss can converge. I did not modify the config and the process. I want to figure out what causes this problem. Thanks!
The train_loss convergence:
![screenshot from 2018-04-23 20-24-30](https://user-images.githubusercontent.com/19221749/39126312-97d998cc-4734-11e8-8b43-dc607fe7eb63.png)
The val_loss convergence:
![screenshot from 2018-04-23 20-27-52](https://user-images.githubusercontent.com/19221749/39126390-d386450a-4734-11e8-89c4-f160c6d183a6.png)

**Comment by Superlee506** [comment]
@whytin In your example, over-fitting happens. You can try the following methods:
1) Add more training  data or data augment;
2) increase the weight decay;
3) simplify  the model

**Comment by whytin** [comment]
I will try, Thanks!
